README.md

```diff
@@ -1,2 +1,186 @@
+* ULTRASONIC SENSOR 
+* ARDUINO UNO  
+* RASPBERRY PI 3  
+* CRYTON MOTOR DRIVER 
+* 2 JOHNSON MOTOR      
+* 2 SERVO MOTOR    
+* SD CARD 
+
+---
+## **Mechanical Aspect of the Design**
+
+### Chassis
+
+
+
+The main body of the FetchBot is 3D printed and comprises the base, a head, and the outer body to cover and contain all the inside components. The shape of the body is hemispherical and the dimensions are 18cm x 18 cm x 9 cm. The body is supposed to be lightweight and rigid.
+
+
+
+### Drive type
+
+
+
+The FetchBot uses a two wheel differential for movement, the idea behind it being that this system will simplify the turn movements as well as rectilinear motion.
+
+
+### Fetching Mechanism
+
+
+
+The FetchBot uses two arms hinges at a place and rotated with a common metal servo motor inside the body with a two gear system. For fetching two mini servo motors are placed at the elbow of the arm. This mechanism helps the bot to fetch and lift the object.
+
+
+![](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/Picture2.png)
+
+---
+
+## **Electronics Aspect of the Design**
+
+### Microcontroller
+
+
+We have used Arduino Uno to control the movement of the motors and get the feedback from the ultrasonic sensor.
+
+### Actuator
+
+
+The robot uses two 12 volt- 100 rpm motors. 
+We also have two servo motors for the sideways movement of the forearms to grab the object and a metal servo motor to rotate the arms together to lift and drop the object
+
+
+![Drive _type](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/arduino%20%26%20motordriver%20connection.jpeg)
+### Motor Driver
+
+We used the Cytron motor driver to control the rotation of our dc motors easily. We preferred it over the L298 motor driver because of the simplicity in connection and coding of the motors.
+
+![Drive _type](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/cytron%20motor%20driver.png)
+
+### Sensor
+We have used an ultrasonic sensor to get realtime feedback of the distance between the object and the bot. We have mounted it in the lower front of the bot to get the best measurements possible.
+
+![Drive _type](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/ultrasonic%20sensor.jpg)
+
+
+## RASPBERRY PI 3 
+Also, we have use Raspberry pi 3 with usb camera to detect QR code which sends feedback to the Arduino Uno.
+
+![Drive _type](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/Picture9.png)
+
+
+### Installing OpenCV and python3 
+Steps to Install OpenCV
+1.Install Python on your system
+2.Install pip
+3.Install OpenCV library using pip
+*  Install Python
+
+*     sudo apt-get install python
+* Install pip
+
+*     sudo apt-get install python3-pip
+* Install OpenCV library with pip
+
+*     pip3 install opencv-python
+ ---
+## QR Code Scanner with OpenCV
```

Comment:
"QR Code Scanner with OpenCV" cannot be a second level heading. Make it `###QR Code Scanner with OpenCV` instead of `##QR Code Scanner with OpenCV`